subsystem/identity_provider_resource_test.go

```diff
@@ -83,6 +83,54 @@ var _ = Describe("Identity provider creation", func() {
 		Expect(terraform.Apply()).To(BeZero())
 	})
 
+	It("Can create a 'gitlab' identity provider", func() {
+		// Prepare the server:
+		server.AppendHandlers(
+			CombineHandlers(
+				VerifyRequest(
+					http.MethodPost,
+					"/api/clusters_mgmt/v1/clusters/123/identity_providers",
+				),
+				VerifyJSON(`{
+				  "kind": "IdentityProvider",
+				  "type": "GitlabIdentityProvider",
+				  "name": "my-ip",
+				  "gitlab": {
+				    "ca": "test-ca",
+				    "url": "https://test.gitlab.com",
+					"client_id": "test-client",
```

Comment:
it seems that the adaptation is not align here and in the next line

Reply:
Fixed this.